Jenkinsfile

```diff
@@ -138,43 +138,16 @@ pipeline {
         stage('Stack 8.0.0-SNAPSHOT SauceLabs') {
```

Comment:
I joined both stages, but maybe there is a non-functional requirement to have two separate stages for visually identifying the stages for merges or PRs.

I think this change makes sense for the sake of maintainability